wikidatarefisland/run.py

```diff
@@ -53,7 +58,7 @@ def main(argv, filepath):
             config.get('blacklisted_item_classes'),
             config.get('ignored_reference_properties')
         )
-        simple_pump.run(item_extractor, args.input_path, args.output_path)
+        dump_reader_pump.run(item_extractor, args.input_path, args.output_path)
```

Comment:
I wonder if using a different flag from `input` would be good here? There is a chance to be confused that every path for `input` and `output` is relative to the data folder except in this new pump.

Reply:
Done